src/asm3/configuration.py

```diff
@@ -346,6 +346,7 @@
     "RecordSearchLimit": "1000",
     "ReloadMedical": "Yes",
     "ReportToolbar": "Yes",
+    "ResizeImagesSpec": "",
```

Comment:
This doesn't need to be here - values that don't exist are empty string by default